corehq/apps/reports/templates/reports/filters/optional_daterange.html

```diff
@@ -2,21 +2,8 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
-{% block filter_js %}
-    {{ block.super }}
-
-    <script>
-        $(function () {
-            bindings = {
-                showFilterName: ko.observable({{ showFilterName|BOOL }})
-            };
-            $('#{{ css_id }}_container').koApplyBindings(bindings);
-        });
-     </script>
-{% endblock %}
-
 {% block filter_content %}
-    <div id="{{ css_id }}_container">
+    <div id="{{ css_id }}_container" class="report-filter-optional-datespan" data-show-filter-name="{{ showFilterName|BOOL }}">
```

Comment:
should the class here just be `report-filter-optional` instead of `report-filter-optional-datespan`?